homeassistant/components/netatmo/light.py

```diff
@@ -31,18 +31,19 @@ async def async_setup_entry(hass, entry, async_add_entities):
 
     data_handler = hass.data[DOMAIN][entry.entry_id][DATA_HANDLER]
 
+    # await data_handler.register_data_class(
+    #     CAMERA_DATA_CLASS_NAME, CAMERA_DATA_CLASS_NAME, None
+    # )
```

Comment:
stale?

Reply:
Nah, I thought I'd save some for later. ;-)